src/connector.ts

```diff
@@ -134,4 +141,33 @@ export class Connector {
 
 		addedChanged(unprotectString(this.coreHandler.core.deviceId))
 	}
+
+	private setupReloadDataKoaEndpoint(): void {
+		const KOA_PORT: number = 3007
+		const RUNDOWN_EXTERNAL_ID_SUFFIX: string = '_1'
+
+		this.koaRouter.post(
+			'/reloadData/rundowns/:rundownName',
+			async (context, next): Promise<void> => {
+				const rundownName: string = context.params.rundownName
+				if (!this.iNewsFTPHandler.iNewsWatcher) {
+					context.status = 503
+					context.response.body = 'Error: iNewsWatcher is undefined'
+					return
+				}
+
+				try {
+					context.response.body = `Attempting to reload rundown with name ${rundownName}`
```

Comment:
The state that is send back should represent the end state of the action. This should be set after the resync action is called.

Reply:
Now correctly does so!